test-app-native-modules.gradle

```diff
@@ -0,0 +1,271 @@
+import groovy.json.JsonSlurper
```

Comment:
Should we move all utility Gradle files into the `android` folder? In general, I think we should only leave the main one in root.